src/libs/antares/study/include/antares/study/parts/hydro/series.h

```diff
@@ -144,10 +144,6 @@ class DataSeriesHydro
     */
     TimeSeries maxHourlyPumpPower;
 
-    // TS's number matrices for Generation and Maximum Power
-    Matrix<uint32_t> timeseriesNumbers;
-    Matrix<uint32_t> timeseriesNumbersHydroMaxPower;
-
     // Equalizing max generation and max pumping numbers of TS's    
     void EqualizeMaxPowerTSsizes(Area& area);
```

Comment:
We still have an **Equalize** function for max power pumping and generation.
We'll have to remove it at some point.

Reply:
See #2068